.circleci/config.yml

```
@@ -56,7 +62,9 @@ workflows:
   version: 2
   pull_request: # Run only for PullRequest
     jobs:
+      
       - house_keeping:
+          context: Dockerhub
```

```suggestion
          context: dockerhub
```
I think the name of the context in svalinn starts with lowercase `d`